client/src/js/services/CashflowService.js

```diff
@@ -0,0 +1,24 @@
+(function (angular) {
+  'use strict';
+
+  /**
+   * @ngdoc service
```

Comment:
The `@ngdoc` symbol is specific to angular's documentation.  It doesn't mean anything for regular jsdoc.